core/src/main/scala/org/apache/spark/ui/UIUtils.scala

```diff
@@ -161,6 +161,7 @@ private[spark] object UIUtils extends Logging {
 
   def commonHeaderNodes: Seq[Node] = {
     <meta http-equiv="Content-type" content="text/html; charset=utf-8" />
+    <link rel="shortcut icon" href="about:blank"/>
```

Comment:
From my understanding this line is less of a fix and more of a hack. If you have front end dev willing to take a look, have them check this too.